akka/src/main/scala/anorm/AkkaStream.scala

```diff
@@ -50,7 +56,7 @@ object AkkaStream {
    * @param m $materializerParam
    * @param connection $connectionParam
```

Comment:
Would be nice to update the Scaladoc to indicate the meaning of the materialized value